resources/templates/docker-compose.yml.tpl

```diff
@@ -0,0 +1,20 @@
+version: '3.8'
+services:
+  gitlab:
+    image: 'gitlab/gitlab-ce:latest'
+    hostname: '${hostname}'
+    environment:
+      GITLAB_OMNIBUS_CONFIG: |
+        external_url 'https://${hostname}'
+        letsencrypt['enable'] = false
+        nginx['ssl_certificate'] = "/etc/gitlab/ssl/${hostname}.crt"
+        nginx['ssl_certificate_key'] = "/etc/gitlab/ssl/${hostname}.key"
+    ports:
+      - '2222:22'
```

Comment:
We should use 22:22 for Gitlab SSH.

Also, please change default SSH port to the EC2 instance, to port 23, for example.

Reply:
Done. The gitlab ssh uses 22:22 and the port to the EC2 instance is 2222